locales/en/create-kafka-instance.json

```diff
@@ -49,21 +50,35 @@
     "warning_or_error_title": "$t(kafka_status_warning_or_error_title)"
   },
   "kafka_status_warning_or_error_title": "This is taking longer than expected.",
+  "learn_about_cloud_regions": "Learn more about cloud regions",
+  "learn_about_sizes": "Learn more about sizes",
   "message_size_value": "up to {{value}} MiB",
   "modal_alerts": {
+    "instance_or_region_unavailable_message": "$t(common:kafka) instances are no longer available. Try again in a few hours.",
+    "instance_or_region_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
     "over_quota_message": "Your organization reached the limit of $t(common:kafka) instances available for creation.",
     "over_quota_title": "$t(common:kafka) instances are unavailable for creation",
     "system_unavailable_message": "$t(common:unexpected_error)",
     "system_unavailable_title": "$t(common:something_went_wrong)",
+    "trial_available_message": "The trial has a duration of 48 hours. If you are interested in using this service in production, go to our <0>Streams for Apache Kafka Overview</0> page to see what options are available during creation and learn how to contact us.",
```

Comment:
Maybe we should parametrize the length of the trial instance as we did for the other limits?